### PR TITLE
Add support for address verification and optional toggle in Program read only view

### PR DESCRIPTION
**Pull request by jessbumblebee**
### Description

Improve the read only version for programs and tests by 
- handling the new address verification properly
- handling the optional toggle better
- adding accessibility check  in tests

There are no mocks for this and I went with what I thought made sense. I'd appreciate a closer look at the screenshots by the reviewers to check if they aggree that having text labels here makes sense and that the text I chose represents the state well. For example I was depating if it should be "required question" or just "required etc. Feedback welcome.

Trackes as one of various improvement suggestions in #4019

## Release notes

N/A hidden behind flag, not user visible

#### General

- [x] Added the correct label: < feature | bug | dependencies | infrastructure | ignore-for-release >
- [x] Created tests which fail without the change (if possible)
- [ n/a ] Extended the README / documentation, if necessary

#### User visible changes

- [x] Wrote browser tests using the [validateAccessibility](https://sourcegraph.com/github.com/civiform/civiform/-/blob/browser-test/src/support/index.ts?L437:14&subtree=true) method
- [x] Manually tested at 200% size
- [x] Manually evaluated tab order

#### New Features

- [x] Add new FeatureFlag env vars to `server/conf/application.conf`
- [x] Conditioned new functionality on a [FeatureFlag](https://docs.civiform.us/contributor-guide/developer-guide/feature-flags)
- [x] Wrote browser tests with the feature flag off and on, etc.

### Issue(s) this completes

n/a

### CODE REVIEW

File: server/app/views/admin/programs/ProgramBlockEditView.java [hunk 1]
```diff
@@ -616,6 +616,19 @@ private DivTag renderQuestion(
               blockDefinition.id(),
               questionDefinition,
               canRemove));
+    } else {
+      // For each toggle, use a label instead in the read only view
+      if (maybeAddressCorrectionEnabledToggle.isPresent()) {
+        String label =
+            addressCorrectionEnabled
+                ? "Address correction: enabled"
```
Comment (dkatzz) [inline server/app/views/admin/programs/ProgramBlockEditView.java:624]: **suggestion (non-blocking):** Optionally can pull this common part out and just use the ternary for the enabled / disabled piece.

**Review by dkatzz — APPROVED** [review]
Changes LGTM - thanks!

**Review by bion — APPROVED** [review]
The UI looks good to me. "Required question" is very clear which IMO is good.

File: server/app/views/admin/programs/ProgramBlockEditView.java (hunk 1 above)
Comment (jessbumblebee) [reply server/app/views/admin/programs/ProgramBlockEditView.java:628]: Done

**Pull request merged**